test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -228,11 +208,9 @@ test.describe( 'Navigation block', () => {
 			} );
 			await addSubmenuButton.click();
 
-			await editor.publishPost();
-
-			await page.locator( 'role=button[name="Close panel"i]' ).click();
+			const postId = await editor.publishPost();
+			await page.goto( `/?p=${ postId }` );
```

Comment:
This was failing for me on trunk. It must have luckily been passing due to the setup of a test before it? It needed to route to the page that was created, not the homepage.

Reply:
These are failing here though, as there are two menus it's resolving to. I guess there's a discrepancy between my local setup and this one. I probably deleted the navigation menu out of the header template in my test setup on accident.